### PR TITLE
Checking the apache error in travis build

### PR DESCRIPTION
**Pull request by nilroy**

### CODE REVIEW

File: recipes/apache.rb [hunk 1]
```diff
@@ -80,12 +80,12 @@
     log_dir: node['platform_family'] == 'rhel' ? '/var/log/httpd' : '/var/log/apache2',
     doc_root: node['platform_family'] == 'rhel' ? '/var/www/html' : '/var/www'
   )
-  notifies :reload, 'service[apache2]'
+  notifies :restart, 'service[apache2]', :immediately
 end
 
 apache_site 'rundeck' do
   enable true
-  notifies :reload, 'service[apache2]'
```
Comment (pcross616) [inline recipes/apache.rb]: would rather not do a restart it will terminate all connections and that can cause automation issues.

**Comment by pcross616** [comment]
Seems that its not detecting systemd vs init.d

`
    * service[apache2] action reload
    * service[apache2]: No custom command for reload specified and unable to locate the init.d script!
    ================================================================================
    Error executing action `reload` on resource 'service[apache2]'
    ================================================================================

`

File: recipes/apache.rb (hunk 1 above)
Comment (nilroy) [inline recipes/apache.rb]: I was just testing stuff. We can reload that

**Comment by nilroy** [comment]
@pcross616 The problem does not come if we test that locally.. I shall close this PR as it some unwanted changes. Will open a new one fresh from the master

**Pull request closed without merging**